internal/bminventory/inventory.go

```diff
@@ -3177,3 +3177,142 @@ func proxySettingsForIgnition(httpProxy, httpsProxy, noProxy string) (string, er
 	}
 	return buf.String(), nil
 }
+
+const liveISOIgnitionConfigFormat = `{
```

Comment:
Looks like this needs to be kept in sync with the onprem-iso-config.ign, which is going to be a problem. Could we just read the same file here?

Reply:
Yes, I've converted the code to read from the same file.